client/components/Payment.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import axios from 'axios';
+import StripeCheckout from 'react-stripe-checkout';
+
+const STRIPE_PUBLISHABLE =
+  process.env.NODE_ENV === 'production'
+    ? 'pk_live_Tqebmm9chI8ljgVt2JPVzyod'
```

Comment:
You need to be cautious about leaving API keys in repos. This should be removed immediately and you should also change your password at this site. If your credit card is on file you may be losing money right now.`.env` is the solution, this should be injected as an environment variable in your webpack build, that is not on this github repo. This repo is public and everyone on the internet can see it.